core/src/main/java/com/linecorp/armeria/common/ContentTooLargeExceptionBuilder.java

```diff
@@ -83,13 +85,24 @@ public ContentTooLargeExceptionBuilder cause(Throwable cause) {
         return this;
     }
 
+    /**
+     * Sets the exception as early rejection.
+     */
+    @UnstableApi
+    public ContentTooLargeExceptionBuilder setEarlyRejection(boolean isEarlyRejection) {
```

Comment:
code style; we don't use `set` in public APIs

```suggestion
    public ContentTooLargeExceptionBuilder earlyRejection(boolean earlyRejection) {
```